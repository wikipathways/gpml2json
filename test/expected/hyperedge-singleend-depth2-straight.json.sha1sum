3e20fa1c1bbeb88738f6e94f6c808e4f906c3e04 ?-
