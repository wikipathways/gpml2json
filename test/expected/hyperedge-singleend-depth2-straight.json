{"pathway":{"@context":["https://cdn.rawgit.com/wikipathways/WpVocabularies/7a46a05/contexts/pvjs.jsonld"],"backgroundColor":"white","borderWidth":0,"color":"black","contains":["idaf905386","id28382a1c","idc088e267","faa53","dde22"],"drawAs":"rect","gpmlElementName":"Pathway","height":270.50460428737574,"kaavioType":"Group","name":"test case","padding":5,"type":["Pathway"],"width":447.7009507346585,"x":0,"y":0,"zIndex":0,"fontSize":12,"fontWeight":"bold","textAlign":"left","verticalAlign":"top","dataSourceVersion":"20130715","standardName":"test case","displayName":"test case","textContent":"Name: test case","isSimilarTo":"http://wikipathways.org/index.php/Special:SearchPathways?query=test%20case&doSearch=1","id":"hash://sha256/53bcd8f34688a1ed273179c962d2926ca0ec60500b6f7c43087bcb835e892b39"},"entityMap":{"faa53":{"id":"faa53","textContent":"Catalyzer","wpType":"GeneProduct","x":195.81847476978646,"y":220.50460428737577,"fontSize":10,"height":20,"verticalAlign":"middle","width":80,"zIndex":32768,"textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","GeneProduct"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"dde22":{"id":"dde22","textContent":"Inhibitor","wpType":"GeneProduct","x":337.7009507346585,"y":182.73984442523766,"fontSize":10,"height":20,"verticalAlign":"middle","width":80,"zIndex":32768,"textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","GeneProduct"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"idaf905386":{"id":"idaf905386","borderWidth":1,"zIndex":12288,"points":[{"x":160.09831016073218,"y":151.36848170556027},{"x":293.43164349406544,"y":151.36848170556024}],"color":"#000000","drawAs":"StraightLine","type":["Interaction","Edge"],"kaavioType":"Edge","gpmlElementName":"Interaction","burrs":["e0874"]},"e0874":{"id":"e0874","attachmentDisplay":{"position":[0.5617283950617287,0],"offset":[0,0]},"drawAs":"none","isAttachedTo":"idaf905386","borderWidth":0,"height":0,"width":0,"zIndex":12288,"type":["Anchor","Burr"],"kaavioType":"Burr","gpmlElementName":"Anchor"},"id28382a1c":{"id":"id28382a1c","borderWidth":1,"zIndex":12288,"points":[{"isAttachedTo":"faa53","attachmentDisplay":{"position":[0.5,0],"offset":[0,0]},"x":235.81847476978646,"y":220.50460428737577,"orientation":[0,-1]},{"isAttachedTo":"e0874","attachmentDisplay":{"position":[0.5617283950617287,0],"offset":[],"relativeOffset":[0,0]},"x":234.99542950229596,"y":151.36848170556024,"orientation":[]}],"color":"#000000","drawAs":"StraightLine","type":["Interaction","Edge"],"kaavioType":"Edge","gpmlElementName":"Interaction","markerEnd":"MimCatalysis","isAttachedTo":["faa53","idaf905386"],"burrs":["e4378"]},"e4378":{"id":"e4378","attachmentDisplay":{"position":[0.4,0],"offset":[0,0]},"drawAs":"none","isAttachedTo":"id28382a1c","borderWidth":0,"height":0,"width":0,"zIndex":12288,"type":["Anchor","Burr"],"kaavioType":"Burr","gpmlElementName":"Anchor"},"idc088e267":{"id":"idc088e267","borderWidth":1,"zIndex":12288,"points":[{"isAttachedTo":"dde22","attachmentDisplay":{"position":[0,0.5],"offset":[0,0]},"x":337.7009507346585,"y":192.73984442523766,"orientation":[-1,0]},{"isAttachedTo":"e4378","attachmentDisplay":{"position":[0.4,0],"offset":[],"relativeOffset":[0,0]},"x":235.48925666279027,"y":192.85015525464956,"orientation":[]}],"color":"#000000","drawAs":"StraightLine","type":["Interaction","Edge"],"kaavioType":"Edge","gpmlElementName":"Interaction","markerEnd":"MimInhibition","isAttachedTo":["dde22","id28382a1c"]}}}
